Automattic/wp-calypso
# Convert package `@automattic/babel-plugin-i18n-calypso` to ESM

#### Changes proposed in this Pull Request

See #55855

#### Testing instructions

- Check all builds are green
- Run `yarn translate` in trunk, and save `build/strings/` and `./calypso-strings.pot`. These are your baseline
- Checkout this branch and run `yarn && yarn translate`. Compare the new `build/strings/` and `./calypso-strings.pot` with the baseline. There should be no differences.
